filebeat/module/haproxy/log/test/tcplog.log-expected.json

```diff
@@ -0,0 +1,31 @@
+[
+    {
+        "@timestamp": "2018-09-20T15:44:23.285Z", 
+        "fileset.module": "haproxy", 
+        "fileset.name": "log", 
+        "haproxy.backend_name": "app", 
+        "haproxy.backend_queue": 0, 
+        "haproxy.bytes_read": 212, 
+        "haproxy.client_ip": "127.0.0.1", 
+        "haproxy.client_port": 40962, 
+        "haproxy.connection_wait_time_ms": -1, 
+        "haproxy.connections.active": 1, 
+        "haproxy.connections.backend": 0, 
+        "haproxy.connections.frontend": 1, 
+        "haproxy.connections.retries": 0, 
+        "haproxy.connections.server": 0, 
+        "haproxy.default.logsource": "127.0.0.1", 
+        "haproxy.frontend_name": "main", 
+        "haproxy.pid": 25457, 
+        "haproxy.process_name": "haproxy", 
+        "haproxy.server_name": "<NOSRV>", 
+        "haproxy.server_queue": 0, 
+        "haproxy.tcp.processing_time_ms": 0, 
+        "haproxy.termination_state": "SC", 
+        "haproxy.total_waiting_time_ms": -1, 
+        "input.type": "log", 
+        "message": "Sep 20 15:44:23 127.0.0.1 haproxy[25457]: 127.0.0.1:40962 [20/Sep/2018:15:44:23.285] main app/<NOSRV> -1/-1/0 212 SC 1/1/0/0/0 0/0", 
```

Comment:
In other modules, message is removed after it is parsed, we may want to do the same on this module.